airflow/providers/cncf/kubernetes/operators/spark_kubernetes.py

```diff
@@ -34,8 +34,9 @@ class SparkKubernetesOperator(BaseOperator):
     :type application_file:  str
     :param namespace: kubernetes namespace to put sparkApplication
     :type namespace: str
-    :param kubernetes_conn_id: the connection to Kubernetes cluster
-    :type kubernetes_conn_id: str
+    :param conn_id: The :ref:`kubernetes connection<howto/connection:kubernetes>`
```

Comment:
```suggestion
    :param kubernetes_conn_id: The :ref:`kubernetes connection<howto/connection:kubernetes>`
```
because the parameter keep the name or you want to change it?

Reply:
Thank you for catching these!